source/assets/scripts/binds/bindSearch.js

```diff
@@ -24,10 +24,16 @@ export async function bindSearch() {
  */
 async function bindHomeSearch() {
   const homeSearchBar = document.getElementById("homepage-search-bar");
+  const homeSearchBarBtn = document.getElementById("homepage-search-btn");
   homeSearchBar.addEventListener("input", (event) => {
     homeSearchBar.textContent = event.target.value;
   });
-  const homeSearchBarBtn = document.getElementById("homepage-search-btn");
+  homeSearchBar.addEventListener("keydown", (event)=>{
```

Comment:
Search on enter implemented

Reply:
Yah, it was lost, ancient code written by the great ancestor Ken Chen